### DISCUSSION TITLE
Can humans in RCareWorld be set to start in a relaxed position?

### DISCUSSION THREAD

**Discussion by nikkijhart** [empriselab/RCareWorld#15] [Help - Programming] [answered]
Is there a way that the humans in RCareWorld can be set to start in their relaxed positions? The joint positions/rotations for the relaxed pose are too complex to input manually into the scene, but it can be inconvenient to wait for the scene to settle once the play button has been hit. Do humans have to start in the initial pose shown in the image?
![image_720](https://github.com/empriselab/RCareWorld/assets/125599638/eb4cad3d-cc80-471f-9332-ba33ff05b027)

**Comment by YoruCathy** [comment, marked as answer]
Yes, if you want the human arms to be articulated soft body. If you want them to be rigid articulated, you can remove the softbody, and it would be faster. If you only care about one arm, then the fastest way is to just add the one arm you are manipulating into the articulation chain, and remove the articulation unit for the rest of the body. You will be able to manually adjust the joints for the rest of the body then.
